<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>761</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="le01"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>에서</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le02"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>까지</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le03"/>
     </item>
     <item>
      <widget class="QPushButton" name="pb">
       <property name="text">
        <string>배수합은</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le04"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>846</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
